content/_snippets/clawback-disclaimer.md

```diff
@@ -0,0 +1 @@
+_Clawback functionality is part of the proposed [XLS-39d-clawback](https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-39d-clawback) extension :not_enabled: to the XRP Ledger protocol. You can use the clawback feature on test networks, but there isn't an official amendment and they aren't available on the production Mainnet. Until there is an amendment, the details documented on these pages are subject to change._ <!-- SPELLING_IGNORE: 39d -->
```

Comment:
As of today, there is an official amendment, Clawback, released in rippled version 1.12.0, so this will have to be updated.

I think this can be changed to just:

> _(Requires the [Clawback amendment](known-amendments.html#clawback))_

Reply:
Thank you, Rome, for your thorough review.